packages/app/src/Element/Note.tsx

```diff
@@ -98,7 +98,14 @@ export default function Note(props: NoteProps) {
   }, [reactions]);
   const positive = groupReactions[Reaction.Positive];
   const negative = groupReactions[Reaction.Negative];
-  const reposts = useMemo(() => dedupeByPubkey(getReactions(related, ev.Id, EventKind.Repost)), [related, ev]);
+  const reposts = useMemo(
+    () =>
+      dedupeByPubkey([
+        ...getReactions(related, ev.Id, EventKind.TextNote).filter(e => e.tags.some(tagFilterOfTextRepost(e, ev.Id))),
```

Comment:
missing `EventKind.TextNote` in `related`

Reply:
I think is fine to only show kind1 reposts in timeline and not as part of note reactions, fetching text notes related to to every note can give us a bunch of results if is part of a thread and we are only interested in a subset (reposts) we can't filter by in the subscription. I think is fine we keep the related filter by kind and not show kind1 reposts here.